README.md

```diff
@@ -278,7 +290,66 @@ Installation Instructions
        callhome_group1: [host-vm1,host-vm2,host-vm3,host-vm4]
        callhome_schedule: [daily,weekly]
      ```
-
+     
+  4. To Install and configure Protocol Service (SMB and NFS) in the cluster you'll need to provide additional information. It is recommended to use [group variables](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#assigning-a-variable-to-many-machines-group-variables) as follows:
+       
+       **IPv4 CES inventory:**
+       ```yaml
+       # group_vars/all.yml:
+       ---
+       scale_protocols:
+           smb: false
+           nfs: false
+           export_ip_pool: [192.168.100.100,192.168.100.101]
+           filesystem: cesSharedRoot
+           mountpoint: /gpfs/cesSharedRoot
+       ```
+     
+       ```yaml
+       smb: set True for required protocol
+       nfs: set True for required protocol
+       export_ip_pool: Comma separated list of ipv6 CES IP
+       filesystem: Any file system that is going to act as cesSharedRoot filesystem
+       mountpoint: CES shared root file system mount point.
+       ```
```

Comment:
Is this the same file as above (group_vars/all.yml), or a different file? Should all of this go into the `scale_protocols` key or are these top-level keys?